conf/fail2ban/postfix-sasl.conf

```diff
@@ -0,0 +1,6 @@
+# Fail2Ban filter for postfix authentication failures
```

Comment:
Cheers ! :+1: 

I think for this fact to be effectively added to the configuration, you should add a similar line to what's here:

https://github.com/YunoHost/yunohost/blob/dev/hooks/conf_regen/52-fail2ban#L16

Reply:
yep that is exactly what I was missing !!

Reply:
thanks